pipelinewise/fastsync/commons/tap_postgres.py

```diff
@@ -152,16 +241,7 @@ def fetch_current_log_pos(self):
         """
         # Create replication slot dedicated connection
         # Always use Primary server for creating replication_slot
-        primary_host_conn_string = "host='{}' port='{}' user='{}' password='{}' dbname='{}'".format(
-            self.connection_config['host'],
-            self.connection_config['port'],
-            self.connection_config['user'],
-            self.connection_config['password'],
-            self.connection_config['dbname']
-        )
-        self.primary_host_conn = psycopg2.connect(primary_host_conn_string)
-        # Set connection to autocommit
-        self.primary_host_conn.autocommit = True
+        self.primary_host_conn = self.get_connection(self.connection_config, prioritize_primary=True)
```

Comment:
🥇 , it connecting to pg was not only duplicated but `'ssl'` option was completely ignored on `primary_host_conn`. Not any more.